src/main/java/com/blamejared/crafttweaker/impl_native/world/ExpandServerWorld.java

```diff
@@ -108,14 +110,22 @@ public static boolean isRaid(ServerWorld internal, BlockPos pos) {
     
     /**
      * Checks if a position is within a chunk that is considered a slime chunk.
-     * 
+     *
      * @param pos The position to look up.
      * @return Whether or not the position was inside a slime chunk.
-     * 
+     *
      * @docParam pos new BlockPos(0, 1, 2)
      */
     @ZenCodeType.Method
     public static boolean isSlimeChunk(ServerWorld internal, BlockPos pos) {
-    	return SharedSeedRandom.seedSlimeChunk(pos.getX() >> 4, pos.getZ() >> 4, internal.getSeed(), 987234911L).nextInt(10) == 0;
+    
+        return SharedSeedRandom.seedSlimeChunk(pos.getX() >> 4, pos.getZ() >> 4, internal.getSeed(), 987234911L)
+                .nextInt(10) == 0;
```

Comment:
Why doesn't this just call `ServerWorld#isSlimeChunk`?

Reply:
You should ask Darkhax.